SPECS/golang/golang.spec

```diff
@@ -117,6 +117,9 @@ fi
 %{_bindir}/*
 
 %changelog
+* Thu Jun 15 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 1.19.10-1
+- Auto-upgrade to 1.19.10 - Adress CVEs
```

Comment:
Should be "Address" and it should list the CVE IDs.

Reply:
Got it. Fixed!